hbp_knowledge/pp2a/torrent2012.bel

```diff
@@ -76,102 +76,138 @@ SET Citation = {"PubMed", "Curr Alzheimer Res. 2012 Feb;9(2):248-56.","22299660"
-act(p(FPLX:PPP2)) -| p(HGNC:MAPT, pmod(Ph,Ser,242))
+act(p(FPLX:PPP2)) -| p(HGNC:MAPT, pmod(Ph,Ser,42))
 UNSET Species
+UNSET Confidence
 
 SET Evidence = "Similar
 findings have been observed in metabolically active rat brain
 slices, where a selective inhibition of PP2A with OA results
 in  an  aberrant  phosphorylation  of  tau  at  the  same  residues
 seen in AD brains at serines (Ser) 198, 199, 202, 396, 404,
 422 and 262 [11, 47, 48]."
+#OA : okadaic acid
+SET Confidence = "Medium"
+SET Species = "10116"
+SET MeSHAnatomy = "Brain"
 a(CHEBI:"okadaic acid") -| act(p(FPLX:PPP2))
-a(CHEBI:"okadaic acid") -> p(HGNC:MAPT, pmod(Ph,Ser,189))
+a(CHEBI:"okadaic acid") -> p(HGNC:MAPT, pmod(Ph,Ser,198))
 a(CHEBI:"okadaic acid") -> p(HGNC:MAPT, pmod(Ph,Ser,199))
 a(CHEBI:"okadaic acid") -> p(HGNC:MAPT, pmod(Ph,Ser,202))
 a(CHEBI:"okadaic acid") -> p(HGNC:MAPT, pmod(Ph,Ser,396))
 a(CHEBI:"okadaic acid") -> p(HGNC:MAPT, pmod(Ph,Ser,404))
 a(CHEBI:"okadaic acid") -> p(HGNC:MAPT, pmod(Ph,Ser,422))
 a(CHEBI:"okadaic acid") -> p(HGNC:MAPT, pmod(Ph,Ser,262))
-p(HGNC:MAPT, pmod(Ph,Ser,189)) pos path(MESH:"Alzheimer Disease")
+p(HGNC:MAPT, pmod(Ph,Ser,198)) pos path(MESH:"Alzheimer Disease")
 p(HGNC:MAPT, pmod(Ph,Ser,199)) pos path(MESH:"Alzheimer Disease")
 p(HGNC:MAPT, pmod(Ph,Ser,202)) pos path(MESH:"Alzheimer Disease")
 p(HGNC:MAPT, pmod(Ph,Ser,396)) pos path(MESH:"Alzheimer Disease")
 p(HGNC:MAPT, pmod(Ph,Ser,404)) pos path(MESH:"Alzheimer Disease")
 p(HGNC:MAPT, pmod(Ph,Ser,422)) pos path(MESH:"Alzheimer Disease")
 p(HGNC:MAPT, pmod(Ph,Ser,262)) pos path(MESH:"Alzheimer Disease")
+UNSET MeSHAnatomy
+UNSET Species
+UNSET Confidence
 
 SET Evidence = "Further experiments based on the
 injection of a PP2A inhibitor in the rat hippocampus demon-
 strated tau hyper-phosphorylation, and learning and memory
 deficits  [49,  50]. "
+SET Confidence = "High"
 SET Species = "10116"
 SET MeSHAnatomy = "Hippocampus"
 a(CHEBI:"okadaic acid") -> p(HGNC:MAPT, pmod(HBP:hyperphosphorylation))
 a(CHEBI:"okadaic acid") -| bp(GO:learning)
 a(CHEBI:"okadaic acid") -| bp(GO:memory)
 UNSET Species
 UNSET MeSHAnatomy
+UNSET Confidence
 
 SET Evidence = "Moreover,  some  tau  kinases  as  cyclin-
 dependent  kinase  5  (cdk5)  and  TPKI  (thiamine  pyrophos-
 phokinase 1)/GSK3 (glycogen synthase kinase 3) are acti-
 vated following PP2A inhibition in starved mice [51]."
+SET Confidence = "High"
 SET Species = "10090"
 act(p(FPLX:PPP2)) -| act(p(HGNC:CDK5))
 act(p(FPLX:PPP2)) -| act(p(HGNC:TPK1))
 act(p(FPLX:PPP2)) -| act(p(HGNC:GSK3B))
 UNSET Species
+UNSET Confidence
 
 SET Evidence = "Several observations showing reduced PP2A activity
 by 30% in the frontal cortex in AD [55], were followed by a
 number of studies of PP2A mRNA and proteins."
+SET Confidence = "Medium"
 SET MeSHAnatomy = "Frontal Lobe"
 path(MESH:"Alzheimer Disease") neg act(p(FPLX:PPP2))
+##Is there any way to annotate the transcipt part?
+#path(MESH:"Alzheimer Disease") neg r(FPLX:PPP2)
```

Comment:
i think this would be the correct way to do so, but in my understanding of the evidence sentence they mentioned only studies that investigate these mRNA and proteins and no real results...

Reply:
Agreed. No results here.